common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3207,6 +3208,15 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(3)
 
+  val CLIENT_FETCH_THROWS_FETCH_FAILURE: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.fetch.throwsFetchFailure")
+      .withAlternative("celeborn.fetch.throwsFetchFailure")
```

Comment:
`withAlternative` is used for backward compatibility. Since this is a new config, we don't need to call `withAlternative`